test/views/api19a/BillViewSpec.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.api19a
+
+import assets.BaseTestConstants._
+import assets.EstimatesTestConstants._
+import assets.FinancialTransactionsTestConstants._
+import assets.IncomeSourceDetailsTestConstants._
+import assets.Messages
+import assets.Messages.{Breadcrumbs => breadcrumbMessages}
+import auth.MtdItUser
+import config.FrontendAppConfig
+import models.calculation.BillsViewModel
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.i18n.Messages.Implicits._
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import play.twirl.api.HtmlFormat
+import utils.ImplicitCurrencyFormatter._
+import utils.TestSupport
+
+class BillViewSpec extends TestSupport {
+
+  val mockAppConfig: FrontendAppConfig = app.injector.instanceOf[FrontendAppConfig]
```

Comment:
There is already an instance of `FrontendAppConfig` made available from `TestSupport`. You could reuse that one instead of defining another one here. Could you also update `EstimatesViewSpec` to do the same please as I missed this in the earlier PR.